contents/blog/analytics-pricing.md

```diff
@@ -0,0 +1,130 @@
+
+## When should I use anonymous events?
+Anonymous events are ideal for tracking users who aren't logged in, or for use cases where you only need to analyze behavior at an aggregate level, such as:
+
+- Visitors to your marketing website
+- Readers of a news website
+- Shoppers on an e-commerce store
+- Users of a B2C mobile app
+- Tracking API and backend events
+- Events generated by LLMs
+- Error monitoring
+
+If you mostly track logged-in users of a B2B product, however, you should continue to do so. 
+
+We're supporting both because we want people to have the level of analysis they need for their use case at a fair price, rather than forcing them to pay more when they only need basic data on individual users.
+
+## Why are anonymous events so much cheaper?
+
+The ELI5 version is that they cost less to ingest and process, so we're passing that saving onto you.
+
+Here's the slightly longer version:
+
+- When you call `posthog.identify`, or an identified user performs an event, we do a bunch of processing to create, update, or merge person profiles in our backend. This is one of the most complex steps in [our ingestion pipeline](/docs/how-posthog-works/ingestion-pipeline), which makes it expensive.
+
+- When ingesting anonymous events, we store data in a sessions table that captures session properties, such as the referring domain and the URLs visited. This table is much cheaper for us to update and query. It's faster, too.  
+
+- This lets us charge you much less for these events, saving you money and enabling many use cases that would have been cost prohibitive before.
+
+## Why are you deliberately making less money?
+
+Because it's the right thing to do for both our users and our business.
+
+Most of our competitors are inefficient. They employ huge [outbound sales teams](/founders/negotiate-software-better) to grow revenue. Their salaries and commissions for closing deals are passed onto their customers through higher prices.
+
+In contrast, we're 100% inbound, we [grow mostly through word of mouth](/handbook/how-we-get-users), and we charge based on actual usage. 
+
+We don't believe in loss-leaders, so we make a modest positive margin on each event sold, but what we charge is directly connected to what it costs us, not what we think we can get away with charging.
+
+We grow our revenue through helping you grow, and onboarding you onto other tools, like session replay, feature flags, and surveys.
+
+We think this is better than trying to squeeze you for every cent you have. You have a great experience, and we'll enjoy better retention and word of mouth for doing the right thing.
+
+> ## FAQ
+>
+> ### Do I still get 1 million free events?
+> es, your first 1 million events each month are free, regardless of what type of events you're sending.
+>
+> ### What are the limitations of anonymous events?
+>
+> When sending anonymous events, you cannot:
```

Comment:
It's not really "when" because you could capture anonymous events now but still have identified events.
```suggestion
> With anonymous events, you cannot:
```